### PR TITLE
Update `AboutUs` page to include photos and roles

### PR DESCRIPTION
**Pull request by david-eom**

### CODE REVIEW

**Review by Zacchua — COMMENTED** [review]
Looks good!

**Comment by david-eom** [comment]
Fixed #5.

File: docs/AboutUs.md [hunk 1]
```diff
@@ -3,57 +3,43 @@ layout: page
 title: About Us
 ---
 
-We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
+## Meet our team!
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+**SociaLite** is a project is built for CS2103T Software Engineering.
```
Comment (stanley-1) [inline docs/AboutUs.md:8]: Remove the word 'is' before "built for CS2103T Software Engineering."

**Review by stanley-1 — COMMENTED** [review]
Thanks for compiling this page! Just a minor typo to fix at the start of the page.

File: docs/AboutUs.md (hunk 1 above)
Comment (david-eom) [reply docs/AboutUs.md:8]: Thanks for the catch! Will remove and commit later on.

**Pull request merged**